livecheck/main.py

```diff
@@ -478,7 +522,7 @@ def main(
     settings = gather_settings(search_dir)
     package_names = sorted(package_names or [])
     for cat, pkg, ebuild_version, version, url, regex, _use_vercmp in get_props(
-            search_dir, settings, package_names, exclude):
+            search_dir, settings, package_names, exclude, progress):
```

Comment:
`progress=progress`

Reply:
Lo siento , no lo entiendo

Reply:
This is in response to the last comment. After you add `*,` the `progress` parameter must always be a keyword argument: `progress=progress`.

Reply:
fix